Dockerfile

```diff
@@ -41,6 +41,6 @@ FROM base as production
 
 RUN bundle install --without development test
 
-RUN PARTNER=graduate RAILS_ENV=production DEVISE_SECRET_KEY=$(bundle exec rails secret) bundle exec rails assets:precompile
+RUN PARTNER=graduate RAILS_ENV=production SECRET_KEY_BASE=$(bundle exec rails secret) bundle exec rails assets:precompile
```

Comment:
I think this should stay as DEVISE_SECRET_KEY, unless that's being set somewhere else.

Reply:
I changed this because in devise.rb, we have the line
  `config.secret_key = yml_devise[Rails.env]['secret_key']`. (It also says that Devise will use the secret_key_base by default).
Also, the CI build was failing because it was telling me that SECRET_KEY_BASE was not set, and this took care of it. But maybe adding `config.secret_key_base = ENV['SECRET_KEY_BASE']` to application.rb will also fix this.

Reply:
Update: adding config.secret_key_base = ENV['SECRET_KEY_BASE'] to application.rb does not remove the message:
ArgumentError: Missing `secret_key_base` for 'production' environment